landing-page/ClientApp/src/app/services/legal-entity-data.service.ts

```diff
@@ -56,7 +56,7 @@ export class LegalEntityDataService extends DataService {
    * Gets the list of security screening records
    */
   getCurrentSecurityScreeningItems(): Observable<SecurityScreeningSummary> {
-    const apiPath = 'api/legalentities/current-security-summary';
+    const apiPath = 'api/legalentities/current-security-summary-new';
```

Comment:
Correct me if I'm wrong but I can't find a new API endpoint that matches the URL - `/api/legalentities/current-security-summary-new`

Perhaps this change was not intended to be committed?